app/controllers/posts_controller.rb

```diff
@@ -38,4 +36,12 @@ def create
       end
     end
   end
+
+  def destroy
+    post = Post.find(params[:id])
+    post.destroy
+
```

Comment:
- [ ] you should Use CanCanCan for Post authorization in Post Controller like below:
```
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end

```

Reply:
Hi @mikethreels, thanks for the review, we would some insight on a problem caused by `load_and_authorize_resource`.

## When we try to create a post, like this:
![image](https://user-images.githubusercontent.com/53308506/167939144-da0dff4b-7727-4cb6-b5a2-c3cb75e57252.png)

## We get the following error:
![image](https://user-images.githubusercontent.com/53308506/167939577-230f8d8d-be32-43bf-917f-b949af7d512b.png)
![image](https://user-images.githubusercontent.com/53308506/167939752-cbb58769-e43d-4929-b106-6ad1f0e7676d.png)

## This is our PostsControllers#create action:
![image](https://user-images.githubusercontent.com/53308506/167940138-6dea41cb-4368-4f79-ada5-a99520fd06ce.png)

Reply:
Hi, we found a [solution](https://github.com/eduardosancho/blog-app/pull/11/commits/5cf0a98160edc2067c52acb5b821cdd509951503), we moved the permitted parameters for the #create action into a private hash.